network-api/networkapi/wagtailpages/templates/wagtailpages/article_page.html

```diff
@@ -11,14 +11,18 @@
 
 {% block content %}
 
+{% if page.hero_layout != "static" %}
```

Comment:
If desired page hero layout is not "static" aka the old template, render the new customizable template